testgrid/cmd/entomologist/BUILD.bazel

```diff
@@ -48,3 +49,9 @@ filegroup(
     tags = ["automanaged"],
     visibility = ["//visibility:public"],
 )
+
+prow_image(
+    name = "image",
+    base = "@git-base//image",
```

Comment:
/hold

Why do these need a git-base image?

Reply:
Please use the default distroless base whenever possible

Reply:
Noted, and done. Thank you